ktor-throughput-benchmark/scripts/profile_benchmark.py

```
@@ -0,0 +1,339 @@
+                text=True,
+                check=True
+            )
+
+            for line in result.stdout.strip().split("\n"):
+                # Look for Gradle worker running our benchmark
+                if "GradleWorkerMain" in line or "ProfileNettyApache" in line or "ProfileBigFile" in line:
+                    # Verify it's actually running our benchmark by checking the process
+                    pid = int(line.split()[0])
+
+                    # Double-check with ps to see command line
+                    ps_result = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)